### DISCUSSION TITLE
User thinks he can enter a search string

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#78513] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

User visits e.g., https://github.com/viking-gps/viking/blob/master/help/C/index.docbook

User places mouse pointer in the code.

User hits CTRL+F or CTRL+G because he wants to search for a string in the code.

![20231210T090834](https://github.com/community/community/assets/1230959/5b8799dc-78b0-42e3-ab4a-48ef90c1b338)

Oddly, the only thing he can do is enter a line number.

And the popup should say "enter only line numbers", else it is boobytrapped, to look like a search box, with ambiguous labeling.

But in fact it should also allow and say one can enter search strings.

So, "great", the browser's CTRL+G is overridden, and all that is possible is a line number search.

No more search within one file.

One can only search within whole trees. 

To do searches for words one must click the pencil to edit the file.

**Comment by Cbodfield** [comment, marked as answer]
Appreciate the feedback @jidanni. We'll be fixing that behavior soon to not override `CTRL+G`. Thanks for bringing it to our attention!
